deploifai/dataset/create.py

```diff
@@ -1,6 +1,8 @@
 import click
 import click_spinner
 from PyInquirer import prompt
+import os
+from time import sleep
```

Comment:
shouldn't require the `time` module at all

Reply:
its used in the spinner part of the code

Reply:
oh ok, my mistake